File: sprint4/By lessons/lesson2code1/lesson2code1.java
// Функция bucket вычисляет номер корзины для указанного фрукта
public static int bucket(String fruit) {
    switch (fruit) {
        case "яблоко":
            return 0;
        case "слива":
            return 1;
        case "груша":
            return 2;
        default:
            throw new IllegalArgumentException("Неизвестный фрукт");
    }
}

File: sprint4/By lessons/lesson2code1/lesson2code1.kt
// функция bucket вычисляет номер корзины для указанного фрукта
fun bucket(fruit: String): Int {
    return when (fruit) {
        "яблоко" -> 0
        "слива"  -> 1
        "груша"  -> 2
        else -> throw IllegalArgumentException("Неизвестный фрукт")
    }
}
